.gitignore

```diff
@@ -1,18 +1 @@
-# Build and Release Folders
-bin-debug/
-bin-release/
-[Oo]bj/
-[Bb]in/
-
-# Other files and folders
-.settings/
-
-# Executables
-*.swf
-*.air
-*.ipa
-*.apk
-
-# Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
-# should NOT be excluded as they contain compiler settings and other important
-# information for Eclipse / Flash Builder.
+/node_modules/
```

Comment:
- [ ] Hello, I can see that you did ignore the `node_modules` directory. However, I think it has been cached because it's still among the files in your repository. 

To fix the cached files issues, navigate into your project from the terminal and run this command `git rm -r --cached .`
After running that command, please stage your files, commit and make a push to GitHub again. Verify that the `node_modules/` directory no longer exists in your repo before asking for a re-review.

Please note that this command `git rm -r --cached .` will clear all cached files and will not push any cached files to your remote repository.